lib/sms.rb

```diff
@@ -0,0 +1,29 @@
+require 'twilio-ruby'
+require 'dotenv'
+
+class Sms
+  attr_reader :credentials
+
+  def initialize
+    @credentials = Dotenv.load
+    @client = twilio_client
+
+  end
+
+  def send(message, number)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `number`. If it's necessary, use `_` or `_number` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)